x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/details.tsx

```diff
@@ -79,6 +81,11 @@ export const Details: React.FC<Props> = memo(({ packageInfo }) => {
     setIsLicenseModalOpen(!isLicenseModalOpen);
   }, [isLicenseModalOpen]);
 
+  const [isChangelogModalOpen, setIsChangelogModalOpen] = useState(false);
+  const toggleChangelogModal = useCallback(() => {
```

Comment:
nit: you can write a toggle without needing any dependency

```suggestion 
  const toggleChangelogModal = useCallback(() => {
    setIsChangelogModalOpen(currentState => !currentState);
  }, []);
```

Reply:
Nice one! thanks Nicolas